userspace/engine/rule_loader.cpp

```diff
@@ -562,10 +562,8 @@ rule_loader::rule_load_exception::~rule_load_exception()
 {
 }
 
-const char* rule_loader::rule_load_exception::what()
+const char* rule_loader::rule_load_exception::what() const noexcept
 {
-	errstr = falco::load_result::error_code_str(ec) + ": "
-		+ msg.c_str();
-
-	return errstr.c_str();
+	// const + noexcept: can't use functions that change the object or throw
+	return msg.c_str();
```

Comment:
This changes rule load exception errors, right?
cc @jasondellaluce i am not sure we want it.

Reply:
I knew it was a deeper change and I was reluctant to do it, but in the end I decided to go ahead because
1. the function has the same name of a virtual function and should have the same signature (i.e. with const + noexcept), this is why a warning was raised by the compiler
2. being noexcept forbids invoking functions that might throw, such as string concatenation
3. I could not find any place in the code where the function is invoked.
If the new implementation is problematic as it changes the original message, let's talk about it.